utils/node_local_test.py

```diff
@@ -1246,7 +1262,8 @@ def create_cont(conf, pool, cont=None, posix=False, label=None, path=None, valgr
 
     rc = _create_cont(conf, pool, cont, posix, label, path, valgrind)
 
-    if rc.returncode == 1 and rc.json['error'] == 'DER_EXIST(-1004): Entity already exists':
+    if rc.returncode == 1 and \
+       rc.json['error'] == 'failed to create container: DER_EXIST(-1004): Entity already exists':
```

Comment:
If this error description has potential to change (probably no often), would it makes sense to do something like this?
```suggestion
       'DER_EXIST(-1004)' in rc.json['error']:
```

Reply:
We shouldn't be changing the json, although it's already changed once hence this update.

This code isn't executed in CI, but rather now that NLT uses labels it'll re-use pools across runs and now sometimes gets this error when creating a container if a previous test run aborted.  I think for safety we could check the status which is just the error code (-1004) as a integer, not string, however I also think this is unlikely to change.

Reply:
> we could check the status which is just the error code (-1004)

For future changes I think this is probably a safer/simpler choice since we don't actually care what the message is in this case.

> this is unlikely to change

Probably not something to worry about then. I.e. no need to push a change just for this.